packages/commons/src/AxeTestEnvironment.cs

```diff
@@ -1,11 +1,33 @@
 namespace Deque.AxeCore.Commons
 {
+    /// <summary>
+    /// Information about the current browser or node application that ran the audit.
+    /// </summary>
     public class AxeTestEnvironment
     {
+        /// <summary>
+        /// User Agent
+        /// </summary>
         public string UserAgent { get; set; }
+
+        /// <summary>
+        /// Window Width.
```

Comment:
It would be nice for this (and WindowHeight) to clarify the unit (presumably pixels?)

Reply:
Looks like it uses https://developer.mozilla.org/en-US/docs/Web/API/Window/innerWidth, which returns values in pixels, Will update the comment for this and WindowHeight.